README.md

```diff
@@ -1,2 +1,37 @@
-# oss-wumbo
-Tracks open source and community activity at the Frontside.
+# What is oss-wumbo?
+
+oss-wumbo is a public repo The Frontside uses to interface with the
+public at a meta level. The purpose of this repo is for you to gain a
+deeper understanding of what open source projects we're currently
+working on at The Frontside.
+
+Our goal is if it pertains to open source we should be working in the
+open and documenting our direction. This is so the users of our
+software can have insight into what is next for the project they're
+relying on. We're always looking for ways to improve how we
+collaborate with the community.  If the users want to, they can
+provide feedback and have an impact on our process or projects.
+
+
+## What is a project champion?
+
+A project champion is a person who is the main manager of an open
+source repo. This person will have the most context of the project and
+documents the roadmap of the project. This does not mean they do all
+the work.
```

Comment:
I would explain why we have the need to have a project champion first. I.e. "unowned projects rot and die of gangrene" and so we need to make sure that there is someone who owns the project and is its advocate both internally and externally. etc... etc..